mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -493,11 +493,22 @@ trait Completions { this: MetalsGlobal =>
         CompletionPosition.CaseKeyword(m.selector, editRange, pos, text, parent)
       case (c: DefTree) :: (p: PackageDef) :: _ if c.namePos.includes(pos) =>
         CompletionPosition.Filename(c, p, pos, editRange)
-      case _ =>
+      case (ident: Ident) :: (t: Template) :: _
+          if isOverrideOrDefIdent(ident) =>
```

Comment:
I'm wondering now if this guard is necessary 🤔 it looks like IntelliJ allows you to type just the name of the method. It's also OK to include any prefix of `override` or `def`

<img width="484" alt="Screenshot 2019-04-06 at 11 39 23" src="https://user-images.githubusercontent.com/1408093/55667743-c3682d80-5860-11e9-9875-52b639fdb188.png">
<img width="473" alt="Screenshot 2019-04-06 at 11 39 29" src="https://user-images.githubusercontent.com/1408093/55667744-c400c400-5860-11e9-83c5-3f46f226dd1b.png">
<img width="534" alt="Screenshot 2019-04-06 at 11 40 01" src="https://user-images.githubusercontent.com/1408093/55667745-c400c400-5860-11e9-8297-81903af723fb.png">


What do you think would be the most natural behavior?